dev/TeachingTip/TeachingTip.cpp

```diff
@@ -958,8 +958,12 @@ void TeachingTip::OnHeroContentPlacementChanged()
 
 void TeachingTip::OnF6AcceleratorKeyClicked(const winrt::CoreDispatcher&, const winrt::AcceleratorKeyEventArgs& args)
 {
-    if (args.VirtualKey() == winrt::VirtualKey::F6)
+    if (!args.Handled() &&
+        IsOpen() &&
+        args.VirtualKey() == winrt::VirtualKey::F6 &&
+        (args.EventType() == winrt::CoreAcceleratorKeyEventType::KeyDown || args.EventType() == winrt::CoreAcceleratorKeyEventType::SystemKeyDown))
```

Comment:
What is the difference between key and systemkey? Why do we check both?

Reply:
Key vs. SystemKey is WM_KEY* vs. WM_SYSKEY*, basically.

I think we should only need KeyDown, not SystemKeyDown.

---
In reply to: [270632961](https://github.com/Microsoft/microsoft-ui-xaml/pull/511#discussion_r270632961) [](ancestors = 270632961)